scss/register.scss

```diff
@@ -0,0 +1,183 @@
+@import "login_reg";
+@import "colors";
+
+/*div.small {
+  width: 50%;
+  float: left;
+  padding: 5px;
+
+
+}
+
+div.big {
+  width: 100%;
+  float: left;
+  padding: 5px;
+
+}
+
+div.placeholder
+{
+  background: white;
+  height: 40px;
+
+}*/
+
+
+
+.passwordStrength
+{
+  &.error
+  {
+    .error{
+      display: inherit;
+    }
+    .strong, .moderate, .weak
+    {
+      display: none;
+    }
+  }
+
+  &.weak
+  {
+    .weak{
+      display: inherit;
+    }
+    .strong, .moderate, .error
+    {
+      display: none;
+    }
+  }
+
+  &.moderate
+  {
+    .moderate{
+      display: inherit;
+    }
+    .strong, .weak, .error
+    {
+      display: none;
+    }
+  }
+
+  &.strong
+  {
+    .strong{
+      display: inherit;
+    }
+    .weak, .moderate, .error
+    {
+      display: none;
+    }
+  }
+}
+
+
+.pwFeedback {
+  margin:10px;
+  font-size:11px;
+  font-weight:600;
+
+  &.error {
+    color: $error;
+  }
+
+  &.weak {
+    color:$weak;
+  }
+
+  &.moderate {
+    color:$moderate;
+  }
+
+  &.strong {
+    color:$strong;
+  }
+}
+
+.strengthIndicator {
+  margin: -5px;
+  width: 200px;
+  padding:5px;
+  height: 20px;
+
+
+  &::before {
+    content: "";
+    display: block;
+    height: 100%;
+  }
+}
+
+#passwordWrapper::after {
+  content: "";
+  display: block;
+  clear: both;
+  float: none;
+}
+
+.error .strengthIndicator {
+  &::before {
+    width: 10%;
+    background-color: $error;
+  }
+}
+
+.weak .strengthIndicator {
+  &::before {
+    width: 30%;
+    background-color: $weak;
+  }
+}
+
+.moderate .strengthIndicator {
+  &::before {
+    width: 60%;
+    background-color: $moderate;
+  }
+}
+
+.strong .strengthIndicator {
+  &::before {
+    width: 100%;
+    background-color: $strong;
+  }
+}
+
+.pwFeedback.error, .pwFeedback.weak, .pwFeedback.moderate, pwFeedback.strong {
```

Comment:
pwfeedback.strong --- syntax fehler 

aktuell ist pwFeedback ein HTML Tag und nicht eine Klasse wie in den anderen